ddtrace/internal/telemetry/writer.py

```diff
@@ -537,12 +549,21 @@ def app_started(self, register_app_shutdown=True):
         if config._config["_sca_enabled"].value() is None:
             self.remove_configuration("DD_APPSEC_SCA_ENABLED")
 
+        products = {
+            product: {
+                "version": _pep440_to_semver(),
+                "enabled": status
+            }
+            for product, status in self._product_enablement.items()
+        }
+
```

Comment:
If we only want to report product change data once can we remove `app-product-change` and only send this products payload in app-started? 

```suggestion
        self._send_product_change_updates = False
```